js/app.js

```diff
@@ -1,23 +1,113 @@
-"use-strict"; 
+"use-strict";
 
-// let totalAnswers=25
-// let currentAnswers=0
+// console.log("hello world");
+//global variables
+let totalAnswers = 25;
+let currentAnswers = 0;
 
-let imgBody = document.getElementById ("img-Body");
-let imgOne = document.getElementById ("img-one");
-let imgTwo = document.getElementById ("img-two");
-let imgThree = document.getElementById ("img-three"); 
+function Product(name) {
+  this.name = name;
+  this.votes = 0;
+  this.views = 0;
+}
+
+Product.prototype.render = function (i) {
+  let img = document.getElementById(`img-${i}`);
+  img.src = `img/${this.name}.jpg`;
+}
+
+let allProducts = [
+  new Product("bag"),
+  new Product("banana"),
+  new Product("bathroom"),
+  new Product("boots"),
+  new Product("breakfast"),
+  new Product("bubblegum"),
+  new Product("chair"),
+  new Product("cthulhu"),
+  new Product("dog-duck"),
+  new Product("dragon"),
+  new Product("pen"),
+  new Product("pet-sweep"),
+  new Product("scissors"),
+  new Product("shark"),
+  new Product("sweep"),
+  new Product("tauntaun"),
+  new Product("unicorn"),
+  new Product("water-can"),
+  new Product("wine-glass"),
+];
+//random image function
+function randImage() {
```

Comment:
Since this function doesn't utilize any kind of image consider naming it differently.